Modules/_cursesmodule.c

```diff
@@ -4732,9 +4732,28 @@ PyInit__curses(void)
         return NULL;
     ModDict = d; /* For PyCurses_InitScr to use later */
 
+    void **PyCurses_API = PyMem_Calloc(PyCurses_API_pointers, sizeof(void *));
```

Comment:
This API is weird. Would you be interested to add a structure of 4 pointers? Something like PyDateTime_CAPI.

The 4 pointers are used by:

```
#define PyCursesWindow_Type (*(PyTypeObject *) PyCurses_API[0])
#define PyCursesSetupTermCalled  {if (! ((int (*)(void))PyCurses_API[1]) () ) return NULL;}
#define PyCursesInitialised      {if (! ((int (*)(void))PyCurses_API[2]) () ) return NULL;}
#define PyCursesInitialisedColor {if (! ((int (*)(void))PyCurses_API[3]) () ) return NULL;}
```

Maybe you can name the 4 members as:

```
PyTypeObject *Window_Type
int SetupTermCalled(void)
int Initialised(void)
int InitialisedColor(void)
```

Or I can merge this PR and then you work on a new PR to add a structure. As you want, tell me what do you prefer ;-)

Reply:
> Or I can merge this PR and then you work on a new PR to add a structure. As you want, tell me what do you prefer ;-)

Oh, I perfer to use another PR to add this structure in this weekend :)